### PR TITLE
CASH-867: Add a more prominent pathway to /lend from /lend/filter

### PR DESCRIPTION
**Pull request by ksekhri**

### CODE REVIEW

File: src/pages/Lend/Filter/LendFilterPage.vue [hunk 1]
```diff
@@ -155,6 +158,8 @@ export default {
 			userId: '',
 			filterMenuOpen: false,
 			selectedCustomCategories: {},
+			// eslint-disable-next-line
+			topMessage: `Welcome to Kiva's new filter page! Take it for a spin below, or <a v-kv-track-event="['Lending', 'click-return-classic', 'Exit-CASH-545-2019']" href="https://www.kiva.org/lend?kexpn=lend_filter.lend_filter_versions&kexpv=c">return to the classic view</a> at any time.`,
```
Comment (ksekhri) [inline src/pages/Lend/Filter/LendFilterPage.vue]: @mcstover will `v-kv-track-event` work here with v-html?

File: src/components/Kv/KvMessage.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<template>
+	<div class="kv-message message-text text-center message-text-confirmation" v-if="safeMessage">
+		<div class="kv-message-row">
+			<img class="kv-message-image" src="~@/assets/images/beta-icon.svg">
+			<div class="kv-message-message">
+				<p class="message" v-html="safeMessage"></p>
```
Comment (mcstover) [inline src/components/Kv/KvMessage.vue]: Please make this a slot instead of using v-html

File: src/pages/Lend/Filter/LendFilterPage.vue (hunk 1 above)
Comment (mcstover) [reply src/pages/Lend/Filter/LendFilterPage.vue]: I think you need to attach an on click handler like we do for clicking Advanced Filters. Clicking on Advanced Filters bubbles up and then fires the "exitLendFilterExp" method from the lend-filter-exp.mixin.js file.

All of those operations also need to be called with your click. In other words you need to do four things:
Update this: `this.lendFilterExpVersion = 'c';`
Call this:	`this.updateLendFilterExp();`
Track your event...
Then perform your redirect.

These first two steps are necessary to ensure the continuity of the opted out state of the experiment.

Comment (ksekhri) [reply src/pages/Lend/Filter/LendFilterPage.vue]: Thanks Matt, missed some steps there

File: src/components/Kv/KvMessage.vue (hunk 2 above)
Comment (ksekhri) [reply src/components/Kv/KvMessage.vue]: Updated :+1:

**Comment by ksekhri** [comment]
Cool @mcstover, this is now updated :+1:

**Pull request merged**